tests/platforms/kubernetes/kubeobj.go

```diff
@@ -77,6 +77,7 @@ func buildDaprAnnotations(appDesc AppDescription) map[string]string {
 			"dapr.io/sidecar-readiness-probe-threshold": "15",
 			"dapr.io/sidecar-liveness-probe-threshold":  "15",
 			"dapr.io/enable-metrics":                    strconv.FormatBool(appDesc.MetricsEnabled),
+			"dapr.io/enable-placement":                  strconv.FormatBool(appDesc.EnablePlacement),
```

Comment:
instead, set "dapr.io/placement-addresses" and set it as empty for disabling it. The code would be much simpler and would be a more extensible solution for future scenarios. Also, we already have a way to set the placement address via daprd, so this would also remove the need for extra param in the daprd binary.

Reply:
I'm not sure if there is an error message missing:
when the user wants to turn on placement, but forgets to configure the address.
If you use address as the basis for judging whether to disable placement, you will lose this error message

Reply:
Actually, we might also want to allow users to skip the installation of the placement service all together.

Reply:
If  also want to use this configuration to skip the installation of the placement service all together, is `dapr.io/enable-placement` more appropriate?

Reply:
Skipping the installation should be done in the Helm Chart, and add a setting like `global.placementService=true/false`.

Reply:
I can understand the idea of ​​replacing `dapr.io/enable-placement` with `dapr.io/placement-addresses` to simplify configuration, but I'm still not sure if users will lose the error message that they forgot to configure addresses. I think users should expect to receive error message feedback if no address is configured when placement is turned on.
Are we going to consider this situation? I'll use `dapr.io/placement-addresses` later if we don't need to consider this situation.
@yaron2

Reply:
@CrazyHZM skipping the installation of the placement service needs to happen at the Dapr installation level. the `dapr.io` annotations targets configuration for a specific deployment. This is where the Helm setting comes into play.

As for the annotation, that's how I see it for `dapr.io/placement-address`:

If no annotation exists, the default value is used (set by the Sidecar Injector).
If an annotation exists (dapr.io/placement-address) with the value `""` for a given deployment, the sidecar will skip connecting to placement.
If an annotation exists (dapr.io/placement-address) with a non empty string value, the sidecar will connect to the configured address.

Thoughts?